velox/connectors/hive/storage_adapters/s3fs/S3FileSystem.cpp

```diff
@@ -222,6 +227,10 @@ class S3Config {
         "hive.s3.iam-role-session-name", std::string("velox-session"));
   }
 
+  int getTranferManagerMaxThreads() const {
+    return config_->get<int>("hive.s3.transfer-manager-max-threads", 25);
```

Comment:
I know the file was already like this, but it would be nice to pull out these literal strings to the top of the file as constants and give them a name (`kS3TranferManagerMaxThreadsConfig` or something similar).

Reply:
We need to move this to HiveConfig. Filed https://github.com/facebookincubator/velox/issues/4300
Should be done in a separate PR.